main.py

```diff
@@ -1,7 +1,54 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
+
+some_actors = [
+    ("George", "Klooney"),
+    ("Kianu", "Reaves"),
+    ("Scarlett", "Keegan"),
+    ("Will", "Smith"),
+    ("Jaden", "Smith"),
+    ("Scarlett", "Johansson")
+]
+
+some_genres = [
```

Comment:
No need "some_" just genres
Also the same with actors.
If it is constants use UPPER_CASE for this